test.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+require_relative 'base_decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
```

Comment:
- [ ] Good job so far but when I run the project from my side nothing is appearing as you can see on the screenshots below ⬇️  Please make sure that Trimmer needs to return this `Maximilan `as mentioned by the previous reviewer and as required for this project 👍 
![ruby](https://github.com/Metaverse-Nabeel/OOP-school-library/assets/6615394/d5a432c1-5b28-43f9-8e5d-eca65cca19d8)

Reply:
Hello @codecaiine 

Now, it is working perfectly fine.